### ISSUE TITLE
Create editable table using v.VuetifyTemplate

### ISSUE THREAD

**Issue by Gabriel-ROBIN** [mariobuikhuizen/ipyvuetify#70]
Like the workaround you have done using VuetifyTemplate in this issue : [Display pandas dataframe as a table](https://github.com/voila-dashboards/voila-vuetify/issues/12)

Is there a way to edit a table using dialogs. I would like to reproduce this feature [https://vuetifyjs.com/en/components/data-tables/#crud-actions](https://vuetifyjs.com/en/components/data-tables/#crud-actions)

Thanks

**Comment by mariobuikhuizen** [comment]
Yes, this can be done with VuetifyTemplate. You can click `< >` on the top right of the linked example to see the Vue code and use that in VuetifyTemplate.

**Comment by luk-f-a** [comment]
I tried using VuetifyTemplate  for the CRUD example. I got it to display but could not get the editing to work. @Gabriel-ROBIN , if you made it work, would you mind sharing the code?

**Comment by luk-f-a** [comment]
if anyone arrives here looking for an editable table example, here's some ugly code of mine https://github.com/luk-f-a/voila-vuetify

**Comment by samiit** [comment]
Thanks @luk-f-a . You helped me much.
Sam
